src/components/SRNEntry.vue

```diff
@@ -104,72 +102,75 @@ export default {
   data() {
     return {
       userIDList: [{ userID: "", valid: false }], //array containing user-id's and a valid flag for each
-      invalidInputMessage: null, //flag to check if the input is in correct format
+      invalidInputMessage: null, // whether the input is in correct format
       isCurrentUserValid: false, // whether the current user is valid
       maxLengthOfSRN: 10,
       validateCount: 0, //this variable tells us how many times the user has been validated.
       invalidLoginMessage: "Please enter correct SRN / कृपया सही SRN दर्ज करें",
-      isLoading: false, 
+      isLoading: false,
     };
   },
   computed: {
-    /** returns length of the list of user IDs */
+    /** Returns length of the list of user IDs */
     userIdListLength() {
       return this.userIDList.length;
     },
-    /** checks if any userID has been entered */
+    /** Checks if any userID has been entered */
     isAnyUserIDPresent() {
       return (
         this.userIDList != undefined &&
         this.userIdListLength > 0 &&
         this.userIDList[0]["userID"] != ""
       );
     },
-    //checks if multiple entries are allowed and if one entry is already inputted
+    /** Whether multiple entries have been made by the user */
     ifUserEnteredMoreThanOne() {
       return !this.isSingleEntryOnly && this.userIdListLength > 1;
     },
-    /** for now, plio does not support multiple input entries */
+    /** For now, plio does not support multiple input entries */
```

Comment:
I don't think this comment goes with the rule `docstrings should tell what the method does/performs`
I think you need to add that as well. The docstring you've added right now is just additional context information, we can keep that in as well.